uiowa/uiowa
# Update dependencies 2022-03

Reviewing a couple of recommended module updates

- photoswipe stable release - https://www.drupal.org/project/photoswipe/releases/3.1.1
- resolves https://github.com/uiowa/uiowa/issues/4230
- closes https://github.com/uiowa/uiowa/pull/4925 address 1.9.0 to [1.10.0](https://www.drupal.org/project/address/releases/8.x-1.10)
- closes https://github.com/uiowa/uiowa/pull/4878 antibot 1.4.0 to [2.0.0](https://www.drupal.org/project/antibot/releases/2.0.0). See additional release notes: [1.5.0](https://www.drupal.org/project/antibot/releases/8.x-1.5)
- closes https://github.com/uiowa/uiowa/pull/4505 entity_usage 2.0.0-beta5 to [2.0.0-beta6](https://www.drupal.org/project/entity_usage/releases/8.x-2.0-beta6)
- closes https://github.com/uiowa/uiowa/pull/4757 imagemagick 3.1.0 to [3.3.0](https://www.drupal.org/project/imagemagick/releases/8.x-3.3). See additional release notes: [3.2.0](https://www.drupal.org/project/imagemagick/releases/8.x-3.2)
- closes https://github.com/uiowa/uiowa/pull/4755 layout_builder_restrictions 2.8.0 to [2.12.0](https://www.drupal.org/project/layout_builder_restrictions/releases/8.x-2.12). See additional release notes: [2.11](https://www.drupal.org/project/layout_builder_restrictions/releases/8.x-2.11), [2.10](https://www.drupal.org/project/layout_builder_restrictions/releases/8.x-2.10), [2.9.0](https://www.drupal.org/project/layout_builder_restrictions/releases/8.x-2.9)
- ~~metatag 1.16.0 to [1.19.0](https://www.drupal.org/project/metatag/releases/8.x-1.19). See additional release notes: [1.18.0](https://www.drupal.org/project/metatag/releases/8.x-1.18), [1.17.0 ](https://www.drupal.org/project/metatag/releases/8.x-1.17)~~ this was already updated https://github.com/uiowa/uiowa/pull/4828
- closes https://github.com/uiowa/uiowa/pull/4607 migrate_plus 5.1.0 to [5.2.0](https://www.drupal.org/project/migrate_plus/releases/8.x-5.2)
- closes https://github.com/uiowa/uiowa/pull/4610 mimemail 1.0.0-alpha3 to [1.0.0-alpha4](https://www.drupal.org/project/mimemail/releases/8.x-1.0-alpha4)
- closes https://github.com/uiowa/uiowa/pull/4899 smart_date 3.4.2 to [3.5.1](https://www.drupal.org/project/smart_date/releases/3.5.1). See additional release notes: [3.5.0](https://www.drupal.org/project/smart_date/releases/3.5.0), [additional beta/release candidates](https://www.drupal.org/project/smart_date/releases), [3.4.3](https://www.drupal.org/project/smart_date/releases/3.4.3)

Avoiding google_analytics and samlauth right now...

## To Test

- Look over the release notes for each module linked above.
- Review https://github.com/uiowa/uiowa#updating-dependencies for testing updates to dependencies locally.
  - Try saving config related to these modules to see if there are config changes not handled by an update hook.
- No db update issues or config import errors
- Sync policy.clas and check /table-of-contents to make sure the views_tree functionality is still functioning.
- Sync grad and check https://grad.uiowa.ddev.site/node/add/thesis_defense to make sure "End" date input is display:none;
- Test a migration against migrate_plus update.
